CaseyShimata/ACHLibrary
# Feature/loanpro 5/ach library has validation classes with methods

Testing instructions:

1. Open the link to post man [Postman - Join Team](https://app.getpostman.com/join-team?invite_code=fe0d6c081b6d248184a93c1ade9660df)
2. Click on Collections > LoanProValidator >  POST ach-data-validate > Send
3. View the resulting JSON including the organized data and all of the data validation error messages. (there is a lot of data, and it will be easier to view if you paste it in a JSON viewer tool like this: [Online JSON Viewer](http://jsonviewer.stack.hu/)).

Verify that the ACHValidationTests for each page, record, and field are present and correct.

Side note:
There are many fields that all have some more business logic type validations rather than purely format considerations. For those I will add validation later. I have not added in the regex to the ACHValidationTestSuite yet. I am going to jump onto the front end and get it connected there before adding in the Regex tests. Adding them should be easy, but I want to have a front end ready to go for sure by tomorrow.
